mappings/net/minecraft/client/particle/AbstractDustParticle.mapping

```diff
@@ -8,4 +8,8 @@ CLASS net/minecraft/class_5736 net/minecraft/client/particle/AbstractDustParticl
 		ARG 8 velocityX
 		ARG 10 velocityY
 		ARG 12 velocityZ
+		ARG 14 parameters
 		ARG 15 spriteProvider
+	METHOD method_33076 addFuzziness (FF)F
```

Comment:
This is more like bleaching the color. The 2nd param is a universal bleach while this method applies extra less (but different) bleaches to each of the 3 color components

Reply:
It actually does the opposite of whitening.